config/connect-kusto-sink.properties

```diff
@@ -1,5 +1,5 @@
+name=AzureKustoSinkConnector
```

Comment:
shouldn't we change the name in readme as well?

Reply:
I will change this back to KustoSinkConnector as shown on the kafka connect list